docs/apache-airflow/start/docker.rst

```diff
@@ -79,6 +79,20 @@ On **Linux**, the mounted volumes in container use the native Linux filesystem u
 
 See:ref:`Docker Compose environment variables <docker-compose-env-variables>`
 
+
+.. warning::
+    Default amount of memory available for Docker on ``MacOS`` is often not enough to get Airflow up and running.
```

Comment:
I am not sure this is the right place for this warning. What do you think to move this to the "Before you begin"  section?

Reply:
Yep. looks better